front/app/containers/Admin/projects/project/analysis/Tags/AutoTaggingModal/index.tsx

```diff
@@ -48,6 +50,9 @@ const AutotaggingModal = ({ onCloseModal }: { onCloseModal: () => void }) => {
       { analysisId, autoTaggingMethod, tagsIds, filters },
       {
         onSuccess: () => {
+          trackEventByName(tracks.autoTaggingWithPredefinedTagsPerformed.name, {
```

Comment:
I don't get the `withPredefinedTags` here, this gets triggered for any auto tagging method, right?

Reply:
Yeah, I think the name was a leftover and I forgot to change it..